.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{js,html,md}]
+max_line_length = 500
```

Comment:
Is there any benefit to this? I would be inclined not to have a line limit, so that each text block is on a single line in all source and translated files.

Reply:
My intention was to avoid the hard wrap configured by some editors. It seems that Intellij works with `max_line_length = off`.